src/main.rs

```diff
@@ -91,6 +92,8 @@ struct BadgeQuery {
     color: Option<String>,
     logo: Option<String>,
     r#type: Option<String>,
+    showLanguage: Option<String>,
+    languageRank: Option<String>,
```

Comment:
The casing here is wrong, it should snake_case.

Reply:
Given that shields.io query parameters usually use camel case (e.g. https://shields.io/badges/lines-of-code), would it be better to use camel case here?

Reply:
Sure, but the Rust code should still be in snake case, and you should use `#[serde(rename)]` for that.

Reply:
Alright then, I've used serde rename.